tests/publiccloud/patch_and_reboot.pm

```diff
@@ -32,6 +32,10 @@ sub run {
 
     ssh_fully_patch_system($remote);
 
+    record_info('UNAME', $args->{my_instance}->ssh_script_output(cmd => 'sudo uname -a'));
+    $args->{my_instance}->ssh_assert_script_run(cmd => 'sudo rpm -qa > /tmp/rpm-qa.txt'));
```

Comment:
Please check if that works as expected. I'm not sure if this works, and if it does, where the file is located (PC instance? openQA instance?)

Reply:
PC instance. I upload it to the openQA in the next line. I will of course check this.

Reply:
Leaving this open until checked.

Reply:
So [here](https://pdostal-server.suse.cz/tests/4460#downloads) you can see that it works. I just remove the `log_name` and `failok` parameters from the `upload_logs()` below the the log name is autogenerated (`patch_and_reboot-rpm-qa.txt`).